com.borgbase.Vorta.yaml

```diff
@@ -0,0 +1,94 @@
+app-id: com.borgbase.Vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+branch: stable
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  - --own-name=org.kde.StatusNotifierItem-4-1
+  # Mounting
+  - --talk-name=org.freedesktop.DBus.*
```

Comment:
This is unacceptable.

Reply:
When I run `flatpak run --log-session-bus com.borgbase.Vorta ` as @bilelmoussaoui suggested, I get this (see below). If you find more restrictive permissions than -`-talk-name=org.freedesktop.DBus.*` and `--talk-name=org.freedesktop.Flatpak.*`, which allow me to still mount borg repositories I am more than happy to here them.

```
C34: -> ca.desrt.dconf call ca.desrt.dconf.Writer.Change at /ca/desrt/dconf/Writer/user
B24: <- :1.30 return from C34
B25: <- :1.30 signal ca.desrt.dconf.Writer.Notify at /ca/desrt/dconf/Writer/user
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.Notifications
C3: -> org.freedesktop.DBus fake AddMatch for org.freedesktop.secrets
C4: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.secrets
C5: -> org.freedesktop.DBus fake AddMatch for org.kde.StatusNotifierItem-4-1
C6: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierItem-4-1
C7: -> org.freedesktop.DBus fake AddMatch for org.kde.StatusNotifierItem-2-1
C8: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierItem-2-1
C9: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.Flatpak
C10: -> org.freedesktop.DBus fake wildcarded AddMatch for com.borgbase.Vorta
C11: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.com.borgbase.Vorta
C12: -> org.freedesktop.DBus fake AddMatch for org.kde.StatusNotifierWatcher
C13: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierWatcher
C14: -> org.freedesktop.DBus fake wildcarded AddMatch for org.KDE
C15: -> org.freedesktop.DBus fake AddMatch for com.canonical.AppMenu.Registrar
C16: -> org.freedesktop.DBus fake GetNameOwner for com.canonical.AppMenu.Registrar
C17: -> org.freedesktop.DBus fake AddMatch for ca.desrt.dconf
C18: -> org.freedesktop.DBus fake GetNameOwner for ca.desrt.dconf
C19: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C20: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C3
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C4
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C5
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C6
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C7
*SKIPPED*
B-1: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C8
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C9
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C10
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C11
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C12
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C13
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C14
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C15
*SKIPPED*
B-1: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C16
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C17
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C18
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C19
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C20
C21: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C22: -> org.freedesktop.DBus fake GetNameOwner for ca.desrt.dconf
C23: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.Flatpak
C24: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.Notifications
C25: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C26: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C27: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
C28: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.secrets
C29: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierItem-4-1
C30: -> org.freedesktop.DBus fake GetNameOwner for org.kde.StatusNotifierWatcher
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C21
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C22
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C23
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C24
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C25
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C26
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C27
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C28
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C29
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C30
*SKIPPED*
C31: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C32: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C33: -> org.freedesktop.Flatpak call org.freedesktop.Flatpak.Development.HostCommand at /org/freedesktop/Flatpak/Development
B-1: <- org.freedesktop.DBus return from C31
B-1: <- org.freedesktop.DBus return from C32
B6: <- :1.67 return from C33
B7: <- :1.67 signal org.freedesktop.Flatpak.Development.HostCommandExited at /org/freedesktop/Flatpak/Development
```
me to still mount borg repositories